packages/flutter/test/widgets/list_view_builder_test.dart

```diff
@@ -261,6 +261,62 @@ void main() {
     callbackTracker.clear();
   });
 
+  testWidgets('ListView.builder 10 items, 2-3 items visible, using prototypeItem', (WidgetTester tester) async {
```

Comment:
This test builds all items in sequence. The idea was to have a test that makes a large jump and then assert that prior to the jump [1,2,3] were build and after the jump [10,11,12] are build and [4,5,6,7,8,9] are never build to prove that using the prototypeExtend is more efficient. Can you modify the test to do something like that?

Reply:
Yes, I changed the test to use 30 items and made a big jump to the end of the `ListView`. I actively check that only the 3 first items have been built (the third is in the cache area) and that only the expected two are visible.

Is this right?